apps/omg_watcher/test/integration/invalid_exit_test.exs

```diff
@@ -0,0 +1,159 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Watcher.Integration.InvalidExitTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: false
+  use OMG.API.Fixtures
+  use OMG.API.Integration.Fixtures
+  use Plug.Test
+  use Phoenix.ChannelTest
+
+  alias OMG.API
+  alias OMG.Eth
+  alias OMG.JSONRPC.Client
+  alias OMG.Watcher.Eventer.Event
+  alias OMG.Watcher.Integration.TestHelper, as: IntegrationTest
+  alias OMG.Watcher.Web.Channel
+
+  @moduletag :integration
+
+  @timeout 40_000
+  @eth API.Crypto.zero_address()
+
+  @endpoint OMG.Watcher.Web.Endpoint
+
+  @tag fixtures: [:watcher_sandbox, :stable_alice, :child_chain, :token, :stable_alice_deposits]
+  test "transaction which is using already spent utxo from exit and happened before end of m_sv causes to emit invalid_exit event ",
```

Comment:
here and in the other place - change `m_sv` to it's full name. Without context this is hard to understand what this constant is